mock/service/node.go

```diff
@@ -203,15 +204,20 @@ func (s *service) NodePublishVolume(
 			ISEphemeral: true,
 		}
 	} else {
-		if req.GetStagingTargetPath() != "" {
-			exists, err := checkTargetExists(req.GetStagingTargetPath())
+		if req.GetTargetPath() != "" {
+			exists, err := checkTargetExists(req.GetTargetPath())
 			if err != nil {
 				return nil, status.Error(codes.Internal, err.Error())
 			}
 			if !exists {
-				status.Errorf(codes.Internal, "staging target path %s does not exist", req.GetStagingTargetPath())
+				// If target path does not exist we need to create the directory where volume will be staged
+				if err = os.MkdirAll(req.TargetPath, os.FileMode(0755)); err != nil {
```

Comment:
I think this should just be mkdir. CO is supposed to create the parent

Reply:
Updated.